pegasus/helpers/section_api_helpers.rb

```diff
@@ -297,7 +297,7 @@ def self.create(params)
       )
     rescue Sequel::UniqueConstraintViolation
       tries += 1
-      retry if tries < 2
+      retry if tries < 3
```

Comment:
No strong opinion, but it might make sense to rename SectionHelpers#random_code to something like SectionHelpers#available_random_code if that happens?